### DISCUSSION TITLE
BeeGFS 7.4.4 and 7.2.14 now available

### DISCUSSION THREAD

**Discussion by philippfalk** [ThinkParQ/beegfs#4] [Announcements]
Dear BeeGFS community,

We are pleased to announce the availability of BeeGFS versions 7.4.4 and
7.2.14. Before upgrading to 7.4.4 from other 7.4.x versions, be sure to read
our release and upgrade notes carefully for more information about a minor
incompatibility between different versions of the metadata services under
certain conditions.

For an overview of the changes in the BeeGFS releases, please refer to the
Release Notes:
https://doc.beegfs.io/7.4.4/release_notes.html
https://doc.beegfs.io/7.2.14/release_notes.html

Please also refer to the Upgrade Guide for information on how to upgrade
from older BeeGFS releases:
https://doc.beegfs.io/7.4.4/advanced_topics/upgrade.html
https://doc.beegfs.io/7.2.14/advanced_topics/upgrade.html

The repository descriptions are available on our Download Page:
https://www.beegfs.io/c/download

Best regards
- Philipp Falk

**Comment by garadar** [comment]
Hi @philippfalk 

Is beegfs-7.4.4  🐝  is compatible with https://linux.mellanox.com/public/repo/mlnx_ofed/latest-23.10/  🌐 ?

https://doc.beegfs.io/latest/release_notes.html#supported-distributions

> The same is true for RDMA driver versions that are generally not problematic on the servers, but can cause issues on the client. The following Mellanox OFED driver versions are supported: 5.8, 5.7, 5.6, 5.5, 5.4, 5.3, 5.2, 5.1, 5.0, 4.9

I'm not sure about the meaning of supported ? 

Best Regards,

**Reply by iamjoemccormick** [reply]
Hi @garadar, 

Supported here means we have run our full integration test suite with those specific MOFED versions and not found any issues.

If you're wanting to use OFED 23.10 with BeeGFS 7.4.4 servers (mgmtd, meta, storage) most probably they are compatible. As the note says though, generally new kernels and OFED versions are not problematic for the servers. Of course if you do find any problems please open an issue!

The one area we sometimes see issues for new kernels and OFED versions is building the client kernel module. However I see client build testing was done with both MOFED 23.10 and 24.1 and no issues reported. That testing was with Rocky 9.3, but I'd expect other distributions to work as well. Note client build testing simply means we verify the client kernel module compiles with that version of the OFED, it is not as comprehensive as our full integration test suite.

**Reply by garadar** [reply]
Thank you ! we will try for our next maintenance.

> Of course if you do find any problems please open an issue!
Certainly !
